ansible/roles/prometheus/templates/prometheus.yml

```diff
@@ -216,18 +216,19 @@ scrape_configs:
         - backend-hel.ooni.org:444
 
   # EC2 instances monitoring: 
-  - job_name: 'ooni-aws-ec2'
+  - job_name: 'ooni-aws-ec2-dev'
```

Comment:
Is it not possible to keep them within the same `job_name`, but differentiate them based on a label?

Reply:
Unfortunately no, job entries requires a different name for each entry and we need a different entry for prod and dev because of the credentials section.

I'm not sure if it's possible to change the job name with relabeling, it doesn't show up in the special labels list that they show here: 
https://grafana.com/blog/2022/03/21/how-relabeling-in-prometheus-works/